src/style.css

```diff
@@ -0,0 +1,134 @@
+@font-face {
+  font-family: Avenir_heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: avenir_book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+}
+
+body {
+  height: 100vh;
+  font-weight: 300;
+}
+
+.search {
+  border: none;
+  outline: none;
+  margin: 17px;
+  display: flex;
+  width: 158px;
+  height: 19px;
+  padding-right: 0;
+  align-items: center;
+  flex: 1;
+}
+
+.search2 {
```

Comment:
use better class name like search-small

also, check this checklist point
<img width="936" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/102411130/8e10c4e6-17cc-4770-b43d-7edebe2c78a0">

Reply:
fix all such cases